tildatorch/main.py

```diff
@@ -0,0 +1,27 @@
+"""This app goes with the Torch Tutorial"""
+
+___name___         = "Tilda Torch"
+___license___      = "MIT"
+___dependencies___ = ["sleep"]
+___categories___   = ["EMF"]
+___bootstrapped___ = False
```

Comment:
minor: this isn't needed, it defaults to False

Reply:
Have removed, and force pushed because i am a naughty